### PR TITLE
[CI][Jenkins] add LTE and CWAG Jenkins pipelines converted from CircleCI

### LINKED ISSUE #3082: [CI/CD][CWF]Migrate CircleCI pipelines to Jenkins
To make CWF test more reliable we need to migrate them to Jenkins infrastructure.

### PR DESCRIPTION
**Pull request by 119Vik**
Signed-off-by: Vitalii Kostenko <vitalii@freedomfi.com>
## Summary

Added Following Jenkins Pipelines converted from the CircleCi
- CWAG integration tests
- LTE integration test

### CODE REVIEW

**Review by mattymo — APPROVED** [review]

File: ci-scripts/JenkinsFile-CWAG-integ-test [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+          } catch (Exception e) {
+            echo "magma-trfserver didn't exists as expected"
+          }
+          try {
+            sh "vboxmanage unregistervm cwag-test --delete"
+          } catch (Exception e) {
+            echo "cwag-test didn't exists as expected"
+          }
+          dir('cwf/gateway') {
+            dir('docker') {
```
Comment (tmdzk) [inline ci-scripts/JenkinsFile-CWAG-integ-test:97]: Question: What's the point to do 2 following dir()  instead of one ? `dir('cwf/gateway/docker')`

**Review by tmdzk — COMMENTED** [review]
LGTM just one tiny question tho @119Vik

File: ci-scripts/JenkinsFile-CWAG-integ-test (hunk 1 above)
Comment (119Vik) [reply ci-scripts/JenkinsFile-CWAG-integ-test:97]: there will be 2 dir statements in any case cause when `dir("docker"){...}` block ends we have few commands executed in  `dir('cwf/gateway')` block.
Other way to do same operations is to do:
```
dir('cwf/gateway'/docker){....}
dir('cwf/gateway'){....}
```

**Comment by 119Vik** [comment]
RestartLTETests

**Pull request merged**